src/toolhouse/sdk.py

```diff
@@ -141,7 +141,13 @@ def get_tools(self, bundle="default"):
         Get Tools
         """
         self.bundle = bundle
-        return self.tools.get_tools(GetToolsRequest(provider=self.provider, metadata=self.metadata, bundle=bundle))
+        res = self.tools.get_tools(GetToolsRequest(provider=self.provider, metadata=self.metadata, bundle=bundle))
+        if bundle != "default" and len(res) == 0:
+            warn(
+                "Warning: get_tools() was called, but the tool bundle does not contain any tools."
+                "Please verify that the bundle includes the required tools before proceeding."
```

Comment:
```
To fix this warning:\n
Solution 1. Go to https://app.toolhouse.ai/bundles, locate the bundle `bundle_name`, then click Edit to add at least one tool.\n
Solution 2. Remove the bundle name from the get_tools() call.
```

Reply:
@iamdaniele Can you check if like this is ok?